container/lxd/lxd.go

```diff
@@ -95,6 +119,169 @@ func (manager *containerManager) Namespace() instance.Namespace {
 	return manager.namespace
 }
 
+// seriesLocalAlias returns the alias to assign to images for the
+// specified series. The alias is juju-specific, to support the
+// user supplying a customised image (e.g. CentOS with cloud-init).
+func seriesLocalAlias(series, arch string) string {
+	return fmt.Sprintf("juju/%s/%s", series, arch)
+}
+
+// seriesRemoteAliases returns the aliases to look for in remotes.
+func seriesRemoteAliases(series, arch string) ([]string, error) {
+	seriesOS, err := jujuseries.GetOSFromSeries(series)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	switch seriesOS {
+	case jujuos.Ubuntu:
+		return []string{path.Join(series, arch)}, nil
+	case jujuos.CentOS:
+		if series == "centos7" && arch == jujuarch.AMD64 {
+			return []string{"centos/7/amd64"}, nil
+		}
+	case jujuos.OpenSUSE:
+		if series == "opensuseleap" && arch == jujuarch.AMD64 {
+			return []string{"opensuse/42.2/amd64"}, nil
```

Comment:
(42.2 was released in Nov 2016, 42.3 July 2017, yet... no way to get it) though not-your-problem for this patch :)